tests/cli_test.py

```diff
@@ -360,33 +344,87 @@ def cleos_abi_file_test():
                 f.write(abi)
                 f.truncate()
 
-        # set the malicious abi
-        cmd = "./programs/cleos/cleos set abi eosio.token " + malicious_token_abi_path
-        processCleosCommand(cmd.split())
-
-        # option '--abi-file' makes token still be transferred from alice to bob after setting the malicious abi
-        cmd = ['./programs/cleos/cleos', '-u', 'http://127.0.0.1:8888', '--print-request', '--abi-file', token_abi_file_arg, 'push', 'action', 'eosio.token', 'transfer', '{ "from":"alice", "to":"bob", "quantity":"25.0000 SYS", "memo":"m" }', '-p', 'alice']
-        outs, errs = processCleosCommand(cmd)
-        assert(b'"/v1/chain/get_raw_abi"' not in errs)
-        cmd = "./programs/cleos/cleos get currency balance eosio.token alice SYS"
-        outs, errs = processCleosCommand(cmd.split())
-        assert(outs.strip().decode('utf-8') == "75.0000 SYS")
-
+        tries = 30
+        while not Utils.arePortsAvailable(set(range(8888, 8889))):
+            Utils.Print("ERROR: Another process is listening on nodeos test port 8888. wait...")
+            if tries == 0:
+                assert False
+            tries -= 1
+            time.sleep(2)
+        nodeId = 'bios'
+        data_dir = Utils.getNodeDataDir(nodeId)
+        assert not os.path.exists(data_dir), 'data_dir exists'
+        os.mkdir(data_dir)
+        walletMgr = WalletMgr(True)
+        walletMgr.launch()
+        node = Node('localhost', 8888, nodeId, cmd="./programs/nodeos/nodeos -e -p eosio --plugin eosio::trace_api_plugin --trace-no-abis --plugin eosio::producer_plugin --plugin eosio::producer_api_plugin --plugin eosio::chain_api_plugin --plugin eosio::chain_plugin --plugin eosio::http_plugin --access-control-allow-origin=* --http-validate-host=false " + "--data-dir " + data_dir + " --config-dir " + data_dir, walletMgr=walletMgr)
```

Comment:
I think this may need `--resource-monitor-not-shutdown-on-threshold-exceeded` & its own `config-dir`

Reply:
Added resource monitor flag.  It has a specified `config-dir`.